### PR TITLE
1033 fix

### PR DESCRIPTION
**Pull request by amber-emmes**
Assign Kits and Kit Assembly pages now automatically trim FedEx barco…des in the confirmation input the same as in the tracking number input, as identified in [1033](https://github.com/episphere/connect/issues/1033).

The Kit Assembly page now also uses different divs for reporting on barcode mismatches and reporting the tracking number source, preventing the package tracking number source from being cleared erroneously.

### CODE REVIEW

File: src/pages/homeCollection/assignKits.js [hunk 1]
```diff
@@ -85,7 +85,14 @@ const assignKitsTemplate = async (name) => {
...
   scannedBarcode2.addEventListener("input", (e) => {
     const scannedBarcodeValue = document.getElementById('scannedBarcode').value.trim();
-    const scannedBarcode2Value = e.target.value.trim();
+    let scannedBarcode2Value = e.target.value.trim();
+    
+    // Trim FedEx tracking numbers (1033)
+  // Already done for the scannedBarcode input in checkTrackingNumberSource
+    if (scannedBarcode2Value.length === 34) {
+      scannedBarcode2Value = scannedBarcode2Value.slice(-12);
+      e.target.value = scannedBarcode2Value;
```
Comment (anthonypetersen) [inline src/pages/homeCollection/assignKits.js:94]: nit - these two lines could be combined to one

**Review by anthonypetersen — APPROVED** [review]

File: src/pages/homeCollection/assignKits.js (hunk 1 above)
Comment (amber-emmes) [reply src/pages/homeCollection/assignKits.js:94]: I use the scannedBarcode2Value later on in the code for cross-reference, so I wanted to set its value and also set e.target.value to update the input itself.

Comment (anthonypetersen) [reply src/pages/homeCollection/assignKits.js:94]: Didn't see that, thanks for the info

File: src/pages/homeCollection/assignKits.js [hunk 2]
```diff
@@ -85,7 +85,14 @@ const assignKitsTemplate = async (name) => {
   scannedBarcode2.onpaste = e => e.preventDefault();
   scannedBarcode2.addEventListener("input", (e) => {
     const scannedBarcodeValue = document.getElementById('scannedBarcode').value.trim();
-    const scannedBarcode2Value = e.target.value.trim();
+    let scannedBarcode2Value = e.target.value.trim();
+    
+    // Trim FedEx tracking numbers (1033)
```
Comment (we-ai) [inline src/pages/homeCollection/assignKits.js:90]: `1033` isn't needed.

**Review by we-ai — APPROVED** [review]

**Review by jhflorey — APPROVED** [review]

**Pull request merged**